staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -94,6 +94,12 @@ func (a *mutatingDispatcher) callAttrMutatingHook(ctx context.Context, h *v1beta
 		return &webhookerrors.ErrCallingWebhook{WebhookName: h.Name, Reason: fmt.Errorf("Webhook response was absent")}
 	}
 
+	for k, v := range response.Response.Annotations {
+		if err := attr.AddAnnotation(h.Name+"/"+k, v); err != nil {
+			return err
```

Comment:
Actually, looking more closely I think this should be:
```
&webhookerrors.ErrCallingWebhook{WebhookName: h.name, Reason: err}
```

This type of error is either ignored or blocking depending on the webhook configuration. It is important to be able to ignore errors to avoid a misconfigured webhook from locking up the cluster.

Reply:
>  or just log the error to the debug log?
Done.

> &webhookerrors.ErrCallingWebhook{WebhookName: h.name, Reason: err}

ErrCallingWebhook is returned for transport-layer errors calling webhooks. It represents a failure to talk to the webhook. So I log the error to the debug log rather than use ErrCallingWebhook